types/vndb/vndb-tests.ts

```diff
@@ -0,0 +1,14 @@
+import VNDB = require('vndb');
+
+async function connect() {
+    try {
+        const vndb: VNDB.vndb = await VNDB.start();
```

Comment:
This seems like an unnecessary namespace:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html

Reply:
Understood. I have removed the unnecessary namespace.